en/components/grids_templates/column-moving.md

```diff
@@ -202,7 +202,7 @@ The last step is to **include** the component mixins with its respective theme:
 ```
 
 > [!NOTE]
-> Depending on the component [**View Encapsulation**](/components/themes/sass/component-themes.html#view-encapsulation) strategy, it may be necessary to `penetrate` this encapsulation using `::ng-deep`
+> Depending on the component [**View Encapsulation**](/products/ignite-ui-angular/angular/components/themes/sass/component-themes.html#view-encapsulation) strategy, it may be necessary to `penetrate` this encapsulation using `::ng-deep`
```

Comment:
@zdrawku @dkamburov These links should be relative to the .md files, so docfx builds the correct link, instead of us hardcoding it. Just FYI. I've merged the PR, but these 3 links should be corrected.

Reply:
Addressed https://github.com/IgniteUI/igniteui-docfx/pull/3524